Cargo.toml

```diff
@@ -14,7 +14,7 @@ tonic = "0.9.2"
 prost = "0.11"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
 tokio-stream = "0.1.14"
-paastech-proto = { git = "https://github.com/paastech-cloud/proto.git" }
+paastech-proto = { git = "https://github.com/paastech-cloud/proto.git", branch = "pomegranate" }
```

Comment:
Perhaps we should not rely on the existence of a non mainline branch of the third party repository, when merging to mainline ?

Reply:
Sure, forgot to update it; pomegranate branch was merged into main anyway !